app/package.json

```diff
@@ -10,7 +10,7 @@
     "3box-comments-react": "0.0.8",
     "@apollo/react-hooks": "^3.1.3",
     "@fvictorio/newton-raphson-method": "^1.0.5",
-    "@gnosis.pm/safe-apps-sdk": "0.4.2",
+    "@gnosis.pm/safe-apps-sdk": "1.0.1",
```

Comment:
Maybe it was implemented before the `contract-proxy-kit` worked as a Safe-app?

Reply:
Hey @germartinez thank you for dropping by and helping out! I really appreciate it :)

`@gnosis.pm/safe-apps-sdk` was originally added as a dependency because the version of `contract-proxy-kit` we were using did not allow access to the underlying `getBySafeTxHash` method that we need [to get transaction details](https://github.com/protofire/omen-exchange/blob/master/app/src/services/cpk.ts#L134).